examples/webhooks/webhooks_test.go

```diff
@@ -0,0 +1,67 @@
+package webhooks
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/moovfinancial/moov-go/pkg/mhooks"
+)
+
+// Example handler func for processing a single event type (AccountCreated)
+func ExampleHandler_SingleEvent() {
+	var handler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {
+		secret := "your-webhook-signing-secret" // fetched from secure storage
+		event, err := mhooks.ParseEvent(r, secret)
+		if err != nil {
+			w.WriteHeader(500)
+			return
+		}
+
+		accountCreated, err := event.AccountCreated()
+		if err != nil {
+			w.WriteHeader(500)
```

Comment:
This will cause the event to be redelivered until it exceeds the maximum number of retries, right? Is that the implied behavior we want people to implement when they receive a webhook they're not handling? 

Is there a status code we could accept which means "ack, but we didn't handle it" - or should that just be a 2xx code?

https://docs.moov.io/guides/webhooks/set-up-webhooks/#respond-within-5-seconds

Reply:
Good point; yeah ideally we don't want folks returning a non-2xx if it's an error where retries won't help.
They should just return 200 in those cases.

Updated the examples [here](https://github.com/moovfinancial/moov-go/pull/113/files)